requirements/base.txt

```diff
@@ -12,6 +12,11 @@ python-heatclient>=1.6.1,<1.7
 python-keystoneclient>=3.17.0,<3.18
 python-novaclient>=7.1.2,<7.2
 tenacity>=6.2
+
+# for hastexo_guacamole_client
 channels>=2.4.0,<3
-pyguacamole>=0.9
+asgiref==3.2.0  # depends on our channels version
+mysqlclient==1.3.14  # keep in sync with edx-platform
+jsonfield2==3.0.3   # keep in sync with edx-platform
```

Comment:
Any chance we can be a little more lenient here, as in using a sensible combination of `>=` and `<`, so as to not break things if edx decides to up the patch level of one of these?

Reply:
Sure, I've gone through the release histories and made some assumptions in 589344 for what should not break things for us _now_ but I'll just have try it out.